<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgoBuilder</class>
 <widget class="QMainWindow" name="AlgoBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>algoBuilder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="changePageButton">
      <property name="text">
       <string>Go To Output Page</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>6</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="ControlWidget" name="controlPage"/>
         <widget class="CreateWidget" name="createPage"/>
         <widget class="mainOutputView" name="outputPage"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart_All"/>
   <addaction name="actionEnd_All"/>
   <addaction name="actionStatus"/>
   <addaction name="actionLoad_Config"/>
   <addaction name="actionLogging"/>
  </widget>
  <action name="actionStart_All">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start All</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="actionLoad_Config">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
  <action name="actionLogging">
   <property name="text">
    <string>Logging</string>
   </property>
  </action>
  <action name="actionEnd_All">
   <property name="text">
    <string>End All</string>
   </property>
   <property name="toolTip">
    <string>End All</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mainOutputView</class>
   <extends>QWidget</extends>
   <header>..mainOutputView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreateWidget</class>
   <extends>QWidget</extends>
   <header>..create.createWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlWidget</class>
   <extends>QWidget</extends>
   <header>..controlWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
